src/components/user-list.html

```diff
@@ -21,5 +18,6 @@
 
         export default UserList;
 
-  </script>
+    </script>
+  </template>
```

Comment:
One more thing 🤓:
The template tag should be a sibling of the `script` tag, not a parent.